### PR TITLE
lec script for synopsys and cadence

### PR DESCRIPTION
**Pull request by davideschiavone**
Add formality script
Updated conformal script
Modified top script to clone automatically the reference/golden model and translated to bash
improved README
tried both on bash and tcsh

### CODE REVIEW

**Comment by davideschiavone** [comment]
@Silabs-ArjanB, @MikeOpenHWGroup - comments?

File: scripts/lec/clone_reference.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#
+#     https://solderpad.org/licenses/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ ! -d "./golden_reference_desing" ]; then
```
Comment (Silabs-ArjanB) [inline scripts/lec/clone_reference.sh:17]: desing -> design (here and in other places in this script)

File: scripts/lec/lec.sh [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [[ -z "${GOLDEN_RTL}" ]]; then
+  echo "The env variable GOLDEN_RTL is empty."
+  echo "Cloning Golden Design...."
+  sh clone_reference.sh
+  export GOLDEN_RTL=$(pwd)/golden_reference_desing/cv32e40p/rtl
```
Comment (Silabs-ArjanB) [inline scripts/lec/lec.sh:21]: desing -> design

**Review by Silabs-ArjanB — CHANGES_REQUESTED** [review]

**Comment by davideschiavone** [comment]
thanks @Silabs-ArjanB

**Comment by davideschiavone** [comment]
@Silabs-ArjanB - can I use Squash and Merge?

**Comment by Silabs-ArjanB** [comment]
> @Silabs-ArjanB - can I use Squash and Merge?

Sure

**Review by MikeOpenHWGroup — APPROVED** [review]

**Pull request merged**